### ISSUE TITLE
PaperDropdownMenu inside PaperDialog

### ISSUE THREAD

**Issue by shoemael** [vaadin/gwt-polymer-elements#61]
I'm seeing some odd behavior when placing a PaperDropdownMenu inside a PaperDialog.  Dynamically added PaperItems appear in the list selection but cannot be selected.  These items are added in a ClickHandler that displays the dialog.

Items that are added outside the handler appear and are selectable. Items added inside the handler appear in the list but don't work.  I'm seeing slightly different things in FireFox(41) vs. Chrome (45.0.2454).  FF displays the entire 6 item list but only the initial 2 items can be selected.  Chrome shows 3-4 items and also only allows the selection of the first two.

Tested with GWT 2.7 and 2.8-snap.  Using vaadin-gwt-polymer-elements-1.1.3.0-alpha1

 An example project is available at: https://github.com/shoemael/polymer-test

**Comment by brOOper** [comment]
Having the same issue. Did you find a way to work around this?

**Comment by shoemael** [comment]
Nope.  I eventually used a standard GWT ListBox and styled it match.  Still hoping for a solution/fix in the future.
